CommonControls/FileTypes/RigidModel/Vertex/Formats/StaticVertex.cs

```diff
@@ -42,6 +43,39 @@ public CommonVertex Read(RmvVersionEnum rmvVersion, byte[] buffer, int offset, i
             return vertex;
         }
 
+        static Vector3 _swapXZ(Vector3 v)
```

Comment:
Should be SwapXY, underscore not used for functions.
Also consider static Vector3 SwapXZ(Vector3 v) => new vector3(...)